fresh/centrifugo-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+    "bundles": {
+        "Fresh\\CentrifugoBundle\\FreshCentrifugoBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "src/": "%SRC_DIR%/"
+    },
+    "env": {
+        "CENTRIFUGO_API_KEY": "%generate(secret)%",
+        "CENTRIFUGO_API_ENDPOINT": "http://localhost:8000/api",
+        "CENTRIFUGO_SECRET": "%generate(secret)%"
```

Comment:
I dont see that these environment variables are used by your bundle's config.

Reply:
here
https://github.com/fre5h/CentrifugoBundle/blob/master/Resources/config/services.yaml#L9
https://github.com/fre5h/CentrifugoBundle/blob/master/Resources/config/services.yaml#L25

Reply:
That is your service definition. I meant your [Configuration](https://github.com/fre5h/CentrifugoBundle/blob/master/DependencyInjection/Configuration.php#L24)

Reply:
I dont see the point of your package being a bundle if you don't. 

Your package could easily be a library instead.

Reply:
My bundle uses DataCollector and adds info into Symfony Profiler Toolbar https://github.com/fre5h/CentrifugoBundle/blob/master/DataCollector/CentrifugoCollector.php
Also it has a little configuration, which can be edited, and it is loaded by Symfony https://github.com/fre5h/CentrifugoBundle/blob/master/DependencyInjection/Configuration.php
Also it registers few services in Symfony DI https://github.com/fre5h/CentrifugoBundle/blob/master/Resources/config/services.yaml, uses autowiring and autobiding and tagged services for registering custom channel authenticators
And also bundle registers several console commands.

I didn't implement it as library, because I wanted a tight integration with Symfony. I wanted to see all logs in Symfony Profiler and it requires to add logs inside http client for Centrifugo.
@Nyholm

Reply:
Sorry if Im unclear. 

My point is: If you wanted to use environment variables for instead of config, there would be no use for this package to be a bundle. The "tight integration" could be achieved with a normal library and a recipe. 

The benefit of a bundle (and I do think you should use a bundle for this) is that you can have configuration and build time validation. 

The current state of your bundle has a major drawback. If I dont define the environment variable `CENTRIFUGO_API_ENDPOINT`, then I would get an error in runtime. If you are using proper configuration for this, I would see the error in buildtime. 

I dont think it is a good idea to merge a recipe that define environment variables that are not properly used. That is why I requested changes to this PR.

Reply:
Ok. I understand